packages/client/wallets/smart-wallet/package.json

```diff
@@ -36,7 +38,7 @@
         "@zerodev/webauthn-key": "5.3.1",
         "permissionless": "0.1.36",
         "uuid": "10.0.0",
-        "viem": "2.17.5"
+        "viem": "2.9.17"
```

Comment:
Hey, I meant the other way around!

You rather than downgrade viem here, you should upgrade viem in `smart-wallet/react` to check the type thing.

This change would break a few aspects of the SDK

Reply:
my bad :sweat_smile:. Upgraded the react app but still didn't fix the type scream :disappointed: